content/docs/1.8.0/advanced-resources/security/volume-encryption.md

```diff
@@ -107,7 +107,12 @@ StorageClass parameters are needed to enable online expansion:
 - `csi.storage.k8s.io/node-expand-secret-name`
 - `csi.storage.k8s.io/node-expand-secret-namespace`
 
+# V2 Volume
+
+- Longhorn will create a `dm` device named `<volume-name>-encrypted` for encrypted V2 volume provided to the workload. This is because a V2 volume requires a `dm` device named `<volume-name>` to manage I/O operations, such as during a V2 volume upgrade.
```

Comment:
Do we need to add the implementation detail?
```suggestion
- Longhorn will create a `dm` device named `<volume-name>-encrypted` for encrypted V2 volume provided to the workload. This is because a V2 volume requires a `dm` device named `<volume-name>` to manage I/O operations, such as during a V2 volume upgrade.
```

Reply:
Removed it. Thanks.